src/format/scan.rs

```diff
@@ -388,7 +418,7 @@ pub(super) fn timezone_offset_2822(s: &str) -> ParseResult<(&str, Option<i32>)>
             Ok((s, None)) // recommended by RFC 2822: consume but treat it as -0000
         }
     } else {
-        let (s_, offset) = timezone_offset(s, |s| Ok(s))?;
+        let (s_, offset) = timezone_offset_internal(s, |s| Ok(s), false, false)?;
```

Comment:
RFC 2822 doesn't say U+2212 is acceptable. But that is because it specifies an ASCII-only email format.

I think it would be helpful if chrono was a little less strict in interpreting the standard, and can also parse RFC 2822 from Unicode sources instead of only bare email.

I'm not sure when Word and similar software automatically replace `-` with U+2212 when used near/between a numbers. But it is not uncommon for typographical reasons.

Reply:
It's a reasonable point. In this case, I lean toward remaining strict about it.